cmd/gardener-admission-controller/app/gardener_admission_controller.go

```diff
@@ -115,6 +116,8 @@ func (o *options) validate() error {
 }
 
 func (o *options) run(ctx context.Context) error {
+	logf.SetLogger(logger.MustNewZapLogger(o.config.LogLevel, o.config.LogFormat))
```

Comment:
What happens when invalid log level or format is specified? Maybe we can add validation to `pkg/admissioncontroller/apis/config/validation/admissioncontrollerconfiguration.go` to prevent invalid values to be configured.